pkg/crc/preflight/preflight_checks_common.go

```diff
@@ -110,17 +110,6 @@ func fixBundleExtracted(bundlePath string, preset crcpreset.Preset) func() error
 		if err := os.MkdirAll(bundleDir, 0775); err != nil {
 			return fmt.Errorf("Cannot create directory %s: %v", bundleDir, err)
 		}
-		if err := validation.ValidateBundle(bundlePath, preset); err != nil {
-			var e *validation.InvalidPath
-			if !errors.As(err, &e) {
-				return err
-			}
-			if bundlePath != constants.GetDefaultBundlePath(preset) {
-				/* This message needs to be improved when the bundle has been set in crc config for example */
-				return fmt.Errorf("%s is invalid or missing, run 'crc setup' to download the bundle", bundlePath)
```

Comment:
And this error message seems better than what we get in ValidateBundlePath?

Reply:
not really following what you try to say 🙄

Reply:
Shouldn't we have this as soon as `bundle` argument is used and tell user the provided path is invalid or missing so either user run it with correct path or or just don't provide any bundle path at all.

Reply:
> And this error message seems better than what we get in ValidateBundlePath?

This removes a relatively clear error message (even if I'm not sure it's even shown to the user): `fmt.Errorf("%s is invalid or missing, run 'crc setup' to download the bundle", bundlePath)`
This is more useful than:
```
$ ./out/linux-amd64/crc setup --bundle=/foo/bar
file '/foo/bar' does not exist
```

Reply:
If that is just related to error message then we can update it to validation function and this error message is not even shown to user (Atleast I didn't find a hack which can use this code flow).

Error can be updated to following as part of validation?
```
fmt.Errorf("%s is invalid or missing, run 'crc setup' to download the bundle", bundlePath)
```

Reply:
I'd say "to download the default bundle"

Reply:
@cfergeau updated